test/Interop/Cxx/class/Inputs/loadable-types.h

```diff
@@ -0,0 +1,65 @@
+#ifndef TEST_INTEROP_CXX_CLASS_INPUTS_LOADABLE_TYPES_H
+#define TEST_INTEROP_CXX_CLASS_INPUTS_LOADABLE_TYPES_H
+
+struct EmptyStruct {};
+
+struct StructWithDefaultConstructor {
+  StructWithDefaultConstructor() {}
+};
+
+struct StructWithAdditionalConstructor {
+  StructWithAdditionalConstructor(){};
+  StructWithAdditionalConstructor(int parameter){};
```

Comment:
```suggestion
  StructWithAdditionalConstructor() {}
  StructWithAdditionalConstructor(int parameter) {}
```

Reply:
This is what `clang-format --style=LLVM` does, though...

Reply:
Ah OK then -- just drop the semicolon then.